cmd/oceantv/broadcast_manager.go

```diff
@@ -127,6 +127,14 @@ func (m *OceanBroadcastManager) CreateBroadcast(
 	if err != nil {
 		return fmt.Errorf("could not update config with transaction: %w", err)
 	}
+
+	// Ensure that the CheckBroadcast cron exists.
+	c := &model.Cron{Skey: cfg.SKey, ID: "Broadcast Check", TOD: "* * * * *", Action: "rpc", Var: projectURL + "/checkbroadcasts", Enabled: true}
```

Comment:
OceanBroadcastManager.CreateBroadcast isn't related to broadcast config creation (which is what creating a broadcast on bench does). OceanBroadcastManager.CreateBroadcast uses the BroadcastService to create a broadcast object on YouTube. In short, this shouldn't be in here but rather in the broadcast save handler for bench.

Reply:
To clarify, I think this should go in "saveBroadcast" in oceanbench

Reply:
In oceanbench, not oceantv's handler of /broadcast/save? Okay I suppose that makes most sense since it's sort of separate to saving a broadcast.

Reply:
Done.